ios/RNTrackPlayer/Models/Track.swift

```diff
@@ -35,6 +36,8 @@ class Track: AudioItem, TimePitching, AssetOptionsProviding {
         guard let url = MediaURL(object: dictionary["url"]) else { return nil }
         self.url = url
         self.headers = dictionary["headers"] as? [String: Any]
+        self.userAgent = dictionary["userAgent"] as? String
+        self.artworkURL = MediaURL(object: dictionary["artwork"])
```

Comment:
Why is this added? Seems unrelated to the user-agent..

Reply:
Oops, I missed this one when rebasing with main 🙃